### PR TITLE
implemented features of deleting and opening a selected tab

### PR DESCRIPTION
**Pull request by Zijing-Liu**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -31,24 +32,67 @@
...
+                  type="checkbox"
+                  id="checkboxTotal"
+                  class="checkbox"
+                  @click="checkAllTabs()"
+                />
+                <label for="checkboxTotal"
+                  >{{ selectedWindow.tabs.length }} Tabs</label
+                >
+              </div>
+
```
Comment (legendword) [inline src/App.vue]:
```suggestion

```

File: src/App.vue [hunk 2]
```diff
@@ -16,6 +16,7 @@
             }"
             @click="windowIndex = ind"
           >
+            <!-- dynamic class -->
```
Comment (legendword) [inline src/App.vue]:
```suggestion

```

File: src/App.vue [hunk 3]
```diff
@@ -122,9 +171,64 @@ export default {
...
+    },
+    deleteTab(windowIndex, tabIndex) {
+      this.windows[windowIndex].tabs.splice(tabIndex, 1);
+      chrome.storage.local.set({ tabs: this.windows[windowIndex].tabs });
+      chrome.storage.local.set({ windows: this.windows });
+      this.windowIndex = this.windows.length > index ? index : index - 1;
+      this.selectedTabIndex = -1;
+    },
+
+    DeleteSelected(windowIndex) {
```
Comment (legendword) [inline src/App.vue]: Please refactor the method name to use camel case

File: src/App.vue [hunk 4]
```diff
@@ -122,9 +171,64 @@ export default {
...
+      });
+    },
+    deleteTab(windowIndex, tabIndex) {
+      this.windows[windowIndex].tabs.splice(tabIndex, 1);
+      chrome.storage.local.set({ tabs: this.windows[windowIndex].tabs });
+      chrome.storage.local.set({ windows: this.windows });
+      this.windowIndex = this.windows.length > index ? index : index - 1;
+      this.selectedTabIndex = -1;
+    },
+
```
Comment (legendword) [inline src/App.vue]:
```suggestion

```

File: src/App.vue [hunk 5]
```diff
@@ -122,9 +171,64 @@ export default {
...
+      }
+    },
+
     formatTime(time) {
       return new Date(time).toLocaleString();
     },
+
+    selectTab() {
+      console.log(this);
+    },
```
Comment (legendword) [inline src/App.vue]:
```suggestion

```

**Review by legendword — CHANGES_REQUESTED** [review]
See my comments on the code for refactoring advices.

File: src/App.vue (hunk 3 above)
Comment (Zijing-Liu) [reply src/App.vue]: resolved

**Review by legendword — APPROVED** [review]
Feature is now working as expected. Merging this now.

**Pull request merged**